kolibri/plugins/learn/assets/src/composables/useCopies.js

```diff
@@ -0,0 +1,21 @@
+import { set } from '@vueuse/core';
+import { reactive } from 'kolibri.lib.vueCompositionApi';
+
+// Shared store
+var displayedCopies = reactive([]);
+
+export default function useCopies() {
```

Comment:
The copies composable that we nixed in the previous PR seems to have crept back in here?

Reply:
Bah - rebasing is a pain I'm sure that's where it came from.